### DISCUSSION TITLE
Windows 10 21390.2025 [Final Release]

### DISCUSSION THREAD

**Discussion by falchion10** [WhatTheBlock/WindowsSimplify#64] [Ideas]
This is the final Insider Preview of Windows 10. This build probably isn't that important to x86 users as they can just use normal Windows 10 builds, but this and a few earlier builds support ARM64. I've tested this build on Apple Silicon hardware and it does indeed work. Your Windows 10 1809 ARM build didn't work for me, quite frankly I didn't know windows ARM versions were even available for 1809. I would like to request that you look into making a debloated build of this specific version for ARM users. The reason I would like this to become a debloated build is because Windows 10 will always be leaner than Windows 11. A debloated build of this Windows 10 version would be lighter than your current Windows 11 build. If you need an ISO of the build just ask and I'll gladly upload one.

**Comment by WhatTheBlock** [comment]
Ok, but 21390 doesn't actually require cumulative updates to be installed. That's just Microsoft testing the update channel, which doesn't bring any improvements or bug fixes.

**Comment by falchion10** [comment]
I didn't ask for cumulative updates to be installed, just wanted to know if an ARM version of this build could happen since its a build of Windows 10 that supports Apple Silicon.

**Reply by WhatTheBlock** [reply]
My last reply just meant that I would make an ARM64 version of 21390.1, not 21390.2025.

**Reply by falchion10** [reply]
Ah, that would be fine. Sorry I didn't realize that.

**Comment by WhatTheBlock** [comment]
If there are any errors after testing, please report back. I don't have an ARM64 device to test, so I can't confirm that it will work.

[21390.1_arm_230207](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w10.arm.230207)

**Reply by falchion10** [reply]
Just downloaded the ISO and tested it. I can't seem to get past the installer, It boots to the installer just fine and shows the Windows logo but after the Windows logo disappears the screen just stays black. I didn't experience this issue when installing the Windows 11 build.

**Comment by WhatTheBlock** [comment]
[21390.1_arm_230207_f](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/w10.arm.230207/21390.1_arm_230207_f.iso)

Try this version. I have replaced the boot.wim back to the original version, at least the system can be installed normally.

**Comment by falchion10** [comment]
Ok, I just tested this new version. I am able to install the OS and the installer works just fine, but when it reboots after the Windows logo, the screen just stays black, just like the Installer did.

**Comment by WhatTheBlock** [comment]
Oh, this is really weird. Maybe I need to buy an ARM64 device and test it.

**Reply by falchion10** [reply]
Oh, that would be unfortunate. I’m sure it’s possible to run Windows ARM on an x86 processor via emulation, or on a phone to test. Do you have an idea of what the issue could potentially be? I’ll gladly stick it out and keep testing builds if you want.

**Comment by WhatTheBlock** [comment]
I will try to use QEMU virtual machine to run ARM64. Android devices are only supported by very few old Qualcomm SoCs, so it is not feasible.

**Reply by falchion10** [reply]
Ah ok, well if you need me to test anymore builds just ask and I’ll gladly test more.

**Comment by WhatTheBlock** [comment]
[19045.2546_arm_230208](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/w10.arm.230207/19045.2546_arm_230208.iso)

**Reply by falchion10** [reply]
Installs just fine, but on first boot-up it gives me the "SYSTEM_THREAD_EXCEPTION_NOT_HANDLED" bsod. Whenever I try to turn it off and on again it just gives me this same bsod.

**Reply by WhatTheBlock** [reply]
Thanks for the test. Looks like NTLite has compatibility issues with Windows 10 ARM64. I'll try another way to process the system image.

**Reply by falchion10** [reply]
No problem, just to note this is the exact same bsod I get when I run you Windows 10 1809 (17763) build you have posted. Not sure why Windows 11 works fine, seems like it's only a Windows 10 issue.

**Comment by falchion10** [comment]
@WhatTheBlock Could you provide me with a vanilla 21390.1 arm64 iso? It appears the download files were removed from Microsoft Update servers, tried using UUPDump and it couldn't retrieve the files.

**Reply by WhatTheBlock** [reply]
ok, wait for me to upload.

![Snipaste_2023-04-12_05-39-20](https://user-images.githubusercontent.com/34786954/231294097-9e566b9a-cf13-49ca-8a35-a78396ba0232.png)

**Reply by falchion10** [reply]
Thanks so much! I somehow lost mine, I guess I deleted it or something. It seems like its really hard to get as well, no one online has the arm64 version.

**Comment by WhatTheBlock** [comment]
### [21390.1.210521-1658.CO_RELEASE_CLIENTPRO_OEMRET_A64FRE_EN-US.part1.rar](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/base/21390.1.210521-1658.CO_RELEASE_CLIENTPRO_OEMRET_A64FRE_EN-US.part1.rar)
### [21390.1.210521-1658.CO_RELEASE_CLIENTPRO_OEMRET_A64FRE_EN-US.part2.rar](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/base/21390.1.210521-1658.CO_RELEASE_CLIENTPRO_OEMRET_A64FRE_EN-US.part2.rar)

**Reply by falchion10** [reply]
Just downloaded it and verified the checksum, everything matches up! Thanks again for the ISO, wanted to see what I could potentially do in NTLite.

**Reply by detesion** [reply]
I would like the combined 21390.2050 image(ISO) file, very grateful.

**Comment by r0ssIV** [comment]
Hi! Do you know if this particular build can be activated with genuine Win10 serial ? It looks like a version with some cutoffs, like  Win Security app not opening etc

**Reply by detesion** [reply]
This version does not require activation and can be used directly. You can't open the Win Security app either because it wasn't available at the time of release, or it could be that this is a preview version and there are some bugs that are causing it.

**Reply by nagarjna** [reply]
Hi, 
Someone please help me in installing CREO software in m1pro with 8gb ram. It's for my university projects so it would be a great help from you all.🤧

**Reply by detesion** [reply]
Using Parallels desktop to run Windows ,then install it on Windows. If you don't know how to use the parallels desktop, then ask and I'll show you the solution!

**Reply by nagarjna** [reply]
@detesion i did installed creo in windows 11 ARM on parallels but the software is not opening due to insufficient RAM
im using m1 pro 8/256 GB

**Comment by MrAlgo** [comment]
Did anyone ever get this to work? I stupidly deleted the original W10 iso that was working with parallels. Tried the part 1 and 2 from here which at least installs fine but then fails to load.

**Reply by MrAlgo** [reply]
Actually, scrap that. Got it to work but don't have network and can't get in to device manager...

![image](https://github.com/WhatTheBlock/WindowsSimplify/assets/30338279/3e06ef76-d4f5-41ee-8dfb-ad1715a1d51f)

**Reply by WhatTheBlock** [reply]
You need to install the virtual machine's network card driver yourself.

**Reply by MrAlgo** [reply]
Thanks for the reply but how without device manager?

**Reply by WhatTheBlock** [reply]
I don't know, but this system image has not been modified.

**Reply by MrAlgo** [reply]
Ok, will investigate.

**Reply by MrAlgo** [reply]
Any idea what drivers it uses?

**Reply by MrAlgo** [reply]
Sorted it by pushing the parallels tools from Mac to W10. Unfortunately, the program I want to run on it fails on install but that's an entirely different issue related to that SW manufacturer.
